### PR TITLE
docs(rate-limiting-advanced): revise parameters descriptions

### PR DESCRIPTION
**Pull request by samugi**
### Summary
Add details to various parameters' descriptions in order to make the documentation more clear and user friendly.

### Reason
This is part of the ongoing effort of improving plugins docs

[KAG-156]

[KAG-156]: https://konghq.atlassian.net/browse/KAG-156?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (kikito) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:76]:
```suggestion
        configured with the value: `path`. Ignored when `config.identifier` has any other value.
```

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (kikito) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:82]:
```suggestion
        configured with the value: `header`. Ignored when `config.identifier` is not `header`.
```

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:12]:
```suggestion
  * Increased performance: Rate Limiting Advanced has better throughput performance with better accuracy. The plugin allows you to tune performance and accuracy via a configurable synchronization of counter data with the backend storage. This can be controlled by setting the desired value on the `sync_rate` parameter.
```

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:15]:
```suggestion
  * More control over which requests contribute to incrementing the rate limiting counters via the `disable_penalty` parameter
```

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:77]:
```suggestion
        configured with the value `path`. Ignored when `config.identifier` has any other value.
```

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:83]:
```suggestion
        configured with the value `header`. Ignored when `config.identifier` is not `header`.
```

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:91]:
```suggestion
        configured to synchronize counter data externally (that is 
```

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:92]:
```suggestion
        `config.strategy` is `cluster` or `redis` and `config.sync_rate` isn't
```

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:183]: Is this saying you can have a host or IP address as the string or that the host must be an IP address?

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (cloudjumpercat) [inline app/_hub/kong-inc/rate-limiting-advanced/_index.md:384]:
```suggestion
        previous hit rates (from the window that immediately precedes the current)
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
Thanks for submitting this PR! I made a few suggestions here and there and had one question that I added.

File: app/_hub/kong-inc/rate-limiting-advanced/_index.md
Comment (samugi) [reply app/_hub/kong-inc/rate-limiting-advanced/_index.md]: The former. I rephrased it to hopefully make it more clear. Please feel free to improve the wording, thank you!

**Review by kikito — APPROVED** [review]

**Comment by kikito** [comment]
This improvement looks good enough as-is IMHO. All the changes apply to the current version of the plugin so there's no need to wait for the next release before we merge this pr in `main`.

**Comment by samugi** [comment]
Hi @cloudjumpercat ! Could you have a look and see if the changes look good? Thanks

**Review by cloudjumpercat — APPROVED** [review]
LGTM!

**Comment by cloudjumpercat** [comment]
@samugi I think this can be merged now even though the plugin ordering doc improvements are set for 3.2, but let me know if this is specific to 3.2 and I can change the base branch before merging!

**Comment by samugi** [comment]
> @samugi I think this can be merged now even though the plugin ordering doc improvements are set for 3.2, but let me know if this is specific to 3.2 and I can change the base branch before merging!

Yes @cloudjumpercat, I can confirm this can be merged now, thank you!

**Pull request merged**
